service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersController.java

```diff
@@ -252,7 +254,33 @@ public void handleSubmittedEvent(
         }
     }
 
-    private List<Element<Direction>> getConfigDirectionsWithHiddenValues() throws IOException {
+    private JudgeAndLegalAdvisor prepareJudge(CaseData caseData) {
```

Comment:
Do we want these methods in the controller, would it not be better for them to be in a service?

Reply:
logic was already there I've just moved it. As this is a bug fix ticket I am not going to move